File: src/main/java/com/ray/Hello.java
package com.ray;

public class Hello {
    public static void main(String[] args) {
//        System.out.println("Hello!");
        String s = "abcde";
        System.out.println(s.charAt(3));
        System.out.println(s.substring(1,4));
        Person p = new Person("Tom",66.5f, 1.7f);
        p.hello();
        System.out.println(p.bmi());
        int score = 88;
        System.out.println(score < 80 || score >90);
        char c = 'A';
        System.out.println(c > 'a');
    }
}

File: src/main/kotlin/com/ray/Hello.kt
package com.ray.com.ray.com.ray

fun main() {
    var s:String? = "abcde"
    s = null
    println(s?.length)
    println(s?.get(3))
    println(s?.substring(3))
    val h = Human(weight = 66.5f, height =  1.7f)
    println(h.bmi())
    val c : Char = 'A'
    println(c.toInt() > 60)     //use Char.code instead
/*    var name:String
    name = "Ray"*/
}

class Human(var name: String = "", var weight: Float, var height: Float) {
    init {
        println("test $weight")
    }
//    constructor(name: String, weight: Float,height: Float):this(weight, height)
    fun bmi():Float{
        val bmi = weight / (height * height)
        return bmi
    }
    fun hello(){
        println("Hello Kotlin")
    }
}